content/docs/command-reference/metrics/diff.md

```diff
@@ -42,9 +42,8 @@ They're calculated between two commits (hash, branch, tag, or any
   no directories among the `targets`, this option is ignored.
 
 - `-t <type>`, `--type <type>` - specify a type of the metric file. Accepted
-  values are: `raw` (default), `json`, `tsv`, `htsv`, `csv`, `hcsv`. It will be
-  used to determine how to parse and format metics for display. See
-  `dvc metrics show` for more details.
+  values are: `raw` (default), `json`. It will be used to determine how to parse
+  and format metics for display. See `dvc metrics show` for more details.
```

Comment:
```suggestion
  and format metics for display. See `dvc metrics show` and `dvc metrics diff`
  for more details.
```
?

Reply:
Actually why not copy this whole last sentence from the add cmd ref?
> It will be saved into the corresponding DVC-file, and used by `dvc metrics show` to determine how to handle displaying metrics.

Reply:
Ah no wait this is already the metrics diff cmd... Sry, so just:
> It will be saved into the corresponding DVC-file, and used to determine how to handle displaying metrics.

Reply:
👍